third-party/rsutils/include/rsutils/concurrency/event.h

```diff
@@ -90,7 +93,7 @@ class event
     {
         std::unique_lock< std::mutex > lock( _m );
         _is_set = false;
-        _is_set = ( std::cv_status::timeout != _cv.wait( lock, timeout ) );
+        _is_set = ( std::cv_status::timeout != _cv.wait_for( lock, timeout ) );
```

Comment:
Same, don't assign `_is_set` here.

Reply:
OK, but note that line 95 still is needed. And this function is not `const`.